src/js/helpers/checkboxTreeHelper.js

```diff
@@ -577,3 +589,66 @@ export const autoCheckImmediateChildrenAfterDynamicExpand = (
 
     return [...new Set([...removeParentPlaceholders, ...newValues])];
 };
+
+export const showAllNodes = (tree) => tree
+    .map((node) => ({
+        ...node,
+        className: '',
+        children: node.children
+            ? node.children
+                .map((child) => {
+                    if (child.children && child.children.some((grand) => grand.className === 'hide')) {
+                        return {
+                            ...child,
+                            className: '',
+                            children: child.children.map((grand) => ({ ...grand, className: '' }))
+                        };
+                    }
+                    return {
+                        ...child,
+                        className: ''
+                    };
+                })
+                .sort(sortNodesByValue)
+            : []
+    }));
+
+export const setNodes = (key, nodes, treeName, cleanNodesFn) => ({
```

Comment:
Wouldn't we want to move these actions to our redux/actions folder keeping consistency throughout our codebase?